backend.Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM fredsimon-docker.jfrog.io/golang:1.13 as BUILDER
+COPY m3util /app/m3util
+COPY model /app/model
+COPY backend/go.mod backend/go.sum /app/backend/
+WORKDIR /app/backend
+RUN go mod download
+COPY . /app
+RUN GOPROXY="https://fredsimon.jfrog.io/fredsimon/api/go/go" GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o main
+
+FROM fredsimon-docker.jfrog.io/alpine:3.12
+WORKDIR /app
+COPY --from=BUILDER /app/backend/main main
```

Comment:
all these lines are duplicated in backend and job, we should consider moving this into a `qsm-base-image`.

Then the Dockerfile will be as simple as this
```
FROM qsm-base-image:1.0
ENTRYPOINT ["./main", "server"]
```